### DISCUSSION TITLE
Scheduled scripts only execute once

### DISCUSSION THREAD

**Discussion by SteenMolberg** [johnlindquist/kit#1186] [Q&A]
I have been trying a scheduled "Stand up and stretch" script, but it only runs when I run it and then one time after that. When I restart scriptkit it runs again once. 

Scriptkit have permissions under Accessibility. 

Anybody knows how I can fix this?

**Comment by johnlindquist** [comment]
@SteenMolberg Can you share the schedule syntax and script so we can test it out?

**Reply by SteenMolberg** [reply]
```
// Name: SampleSchedule
// Schedule: */5 * * * *

import "@johnlindquist/kit"

notify(`Timer ⏰`)
```
More and less same as your sample :)
If I restart the script watcher the script run again one time.
Maybe a complete reinstall?

**Reply by johnlindquist** [reply]
Can you make sure "terminal-notifier" has permissions?

![image](https://user-images.githubusercontent.com/36073/230778468-d669c639-789b-41a7-99b8-3469ea2673d5.png)

(that's the "node-notifier" app bundled with Script Kit to send notifications)

**Reply by SteenMolberg** [reply]
Yes it's got that already and the notification does pop up one time after restart og when I run manually.

When I "Restart the script watcher" the script is in fact marked with "Next: 4 minutes... ". I can also view the script from Api > Scheduled scripts. But when the message pops op once I don't see the "Next: " message anymore. 

When I try to run the script the notification pops up right away, but it does not get "scheduled".

**Reply by johnlindquist** [reply]
On my phone, but can you try something like this instead of "notify" to make sure it's not a notify issue:

let filePath = home("testing-schedule.txt")
await appendFile(filePath, Date.now().toString())

**Reply by johnlindquist** [reply]
Sorry, added the "toString()"

**Reply by johnlindquist** [reply]
Got a chance to test it on my laptop. Unfortunately, it is broken in the latest build. After the scheduled script runs, the "remove process" event is unscheduling any future runs. I'll have it fixed this week. Thanks for brining it to my attention.

**Reply by SteenMolberg** [reply]
Hi John.. nice to know that I did not do anything wrong 😅 Looking forward for a fix
